website/src/pages/log-streaming/configure-streaming-for-third-party-tools/index.mdx

```diff
@@ -92,6 +90,17 @@ $ $KAFKA_HOME/bin/connect-distributed.sh $KAFKA_HOME/config/connect-distributed.
 
 ### Install Splunk
 
+Typically, those interested in this functionality already have Splunk implemented in their environment and have the technical expertise to configure it with the Kafka messaging services.  
```

Comment:
I'd delete this entire statement, even if the first statement is true, I think that the second one is really the point of this guide. Users that already know how to publish events in a Kafka topic to Splunk would have stopped after they had set up log streaming from IBM Log Analysis or Activity Tracker to Event Streams.